python/src/aiconfig/editor/client/src/components/aiconfigReducer.ts

```diff
@@ -192,6 +199,16 @@ export default function aiconfigReducer(
         },
       }));
     }
+    case "SET_NAME_DESCRIPTION": {
```

Comment:
Out of curiosity, why did you bundle name and description together in a single action? (It is fine as is)

Reply:
Will update to do separately (to match endpoints), but was keeping them together to start in case we wanted to do something similar to workbooks where we save the values once the title/description component toggles from 'edit' mode to 'read' mode (instead of each change).